### PR TITLE
Add documentation for using PLS with Neovim

### PR DESCRIPTION
**Pull request by WhoIsSethDaniel**

### CODE REVIEW

**Comment by WhoIsSethDaniel** [comment]
ref: #48

**Comment by FractalBoy** [comment]
The way you documented it implies that those configuration settings aren't available in Neovim.

There should probably be two separate `setup` sections - one for VSCode (just install the extension) and one for Neovim (your instructions)

**Comment by WhoIsSethDaniel** [comment]
I'll fiddle with it later today and resubmit. Thanks.

**Comment by FractalBoy** [comment]
Also, please update the POD to mention that PLS has been tested on Neovim as well:

https://github.com/WhoIsSethDaniel/perl-language-server/blob/b7927280209565bc74bae4d7fcb986628dc42c2f/server/lib/PLS.pm#L79-L82

**Comment by WhoIsSethDaniel** [comment]
I updated the POD. I added a Setup section that includes the steps specific to  VSCode and Neovim.

File: README.md [hunk 1]
```diff
@@ -20,8 +20,49 @@ The features currently implemented are:
...
+
+## Setup
+
+### VSCode
+
+Install the fractalboy.pls extension in Visual Studio Code: https://marketplace.visualstudio.com/items?itemName=FractalBoy.pls
+
+### Neovim
+
+This assumes Neovim 0.5.0 or greater and that you have [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig) installed.
```
Comment (FractalBoy) [inline README.md:33]:
```suggestion
This assumes Neovim 0.5.0 or greater.

Install [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig).
```

**Review by FractalBoy — CHANGES_REQUESTED** [review]
Looking better - one more small suggestion.

**Comment by WhoIsSethDaniel** [comment]
Added suggested change.

**Review by FractalBoy — APPROVED** [review]

**Pull request merged**